graders/python/python_autograder/code_feedback.py

```diff
@@ -467,12 +488,24 @@ def bad(msg: str) -> Literal[False]:
         return True
 
     @classmethod
-    def call_user(cls, f: Callable, *args: Any, **kwargs: Any) -> Any:
+    def call_user(
+        cls,
+        f: Callable,
+        *args: Any,
+        stop_on_exception: bool = False,
```

Comment:
Only slight nitpick is that I don't think instructors will be able to use the parameter name `stop_on_exception` as a keyword argument to a student called function. Probably not a big deal, and if we think this will come up then we can always add a leading underscore or similar to the argument name.

Reply:
Done here https://github.com/PrairieLearn/PrairieLearn/pull/10590/commits/d7a6f35f20af13feef4fc06b48c6603212a2f124